localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ maldraxxi_adj:0 "Maldraxxi"
+ maldraxxi_adherent:0 "Maldraxxi"
+ maldraxxi_adherent_plural:0 "Maldraxxi"
+ maldraxxi_desc:0 "A death cult drawing powers from the dimension they call Maldraxxus. Their necromancers are known for creating abominations, animated creatures comprised of multiple dead limbs and body parts."
+ 
+ throsic:0 "Throsi"
+ throsic_adj:0 "Throsic"
+ throsic_adherent:0 "Throsic"
+ throsic_adherent_plural:0 "Throsi"
+ throsic_desc:0 "Many spirits of the fallen Drust have been banished into an offshoot plane within the Emerald Nightmare that they later named Thros, where they cultivated a potent form of death magic with bastardized druidic elements. This form of magic resulted from numerous attempts to bewitch the souls of fallen Drust into constructs to use against human settlers. Following this, the Throsic form of necromancy prefers binding the souls of the dead into durable constructs, while the traditional form of corpse reanimation is rarer. Its influence towards the wildlife and the living is usually perpetuated by the local witch covens. The most loyal of Throsian witches can also ascend into Matrons, an incorporeal banshee-like creature, through a ritualistic act of slitting their own throat in ultimate devotion to their horrific masters."
+ 
+ helya:0 "Helism"
+ helya_adj:0 "Helic"
+ helya_adherent:0 "Helist"
+ helya_adherent_plural:0 "Helists"
+ helya_desc:0 "Helya, the matriarch of Helheim, which is the underworld of the Vrykul who have fallen and haven't ascended to the Halls of Valour, was a Watcher who assisted Odyn in imprisoning elementals into the Elemental Plane. She would be the first spectral Val'kyr when she rebelled against Odyn. She would soon create a champion who would bear Odyns Warswords of the Valajar, which Helya stole from Odyn. This champion would slay many for their souls to serve in Helheim. She would also create the Helarjar, a holy order mocking Odyn's Valarjar. The Kvaldir of Helya would also be responsible for murdering those of dishonour and unworth, taking them to Helheim."
+ 
+ pandaren_religion_group:0 "Chi"
+ pandaren_religion_group_adj:0 "Chi"
+ pandaren_religion_group_adherent:0 "Chiist"
+ pandaren_religion_group_adherent_plural:0 "Chiists"
+ pandaren_religion_group_desc:0 "The philosophy of Chi is based on two things: overcoming negative emotions and spiritual ascension. Both goals are achieved through meditation and self-control."
+ 
+ august_celestials:0 "Celestism"
+ august_celestials_adj:0 "Celestial"
+ august_celestials_adherent:0 "Celestist"
+ august_celestials_adherent_plural:0 "Celestists"
+ august_celestials_desc:0 "The August Celestials are four wild gods of Pandaria. The celestials built temples to train monks in their respective styles after the pandaren revolution. Many native races of Pandaria respect these creatures and worship them as gods. The Celestials teach to overcome negative emotions and develop a spiritual connection with the world through meditations."
```

Made a addition to the last sentence.
```suggestion
 august_celestials_desc:0 "The August Celestials are four wild gods of Pandaria. The celestials built temples to train monks in their respective styles after the pandaren revolution. Many native races of Pandaria respect these creatures and worship them as gods. The Celestials teach their followers to overcome negative emotions and develop a spiritual connection with the world through meditations."
```